### ISSUE TITLE
TypeScript changes not deployed, even though ng serve detects the changes

### ISSUE THREAD

**Issue by etovian** [angular/angular-cli#5600]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.
1. Make a change to a TypeScript file
2. Observe that webpack recompiles
3. Observe that the browser refreshes
4. Observe that the TypeScript changes were not deployed to the browser
5. Make a change to an html template file
6. Observe that the html changes have been deployed to the browser, but any TypeScript changes have not

### The log given by the failure.
No exceptions are thrown

### Mention any other details that might be useful.
IDE: IntelliJ
OS: Windows 10

**Comment by etovian** [comment]
It's probably also pertinent that I'm running _ng serve_ with the --proxy-config switch.  Here's my config file:

{
  "/api": {
    "target": "http://localhost:8080",
    "secure": false
  }
}

**Comment by etovian** [comment]
But the problem persists, even if I run _ng serve_ without the --proxy-config switch.  In either case, if I halt _ng serve_ and run it again, the latest TypeScript changes are deployed to the browser after it refreshes.

**Comment by yarrgh** [comment]
Which version of the cli are you using?

**Comment by etovian** [comment]
Sorry, I thought I included ng --version output in the ticket:

@angular/cli: 1.0.0-rc.4
node: 6.10.0
os: win32 x64
@angular/common: 2.4.10
@angular/compiler: 2.4.10
@angular/core: 2.4.10
@angular/forms: 2.4.10
@angular/http: 2.4.10
@angular/material: 2.0.0-beta.2
@angular/platform-browser: 2.4.10
@angular/platform-browser-dynamic: 2.4.10
@angular/router: 3.4.10
@angular/cli: 1.0.0-rc.4
@angular/compiler-cli: 2.4.10

**Comment by clydin** [comment]
This PR should hopefully address the issue: https://github.com/angular/angular-cli/pull/5492

**Comment by filipesilva** [comment]
@etovian are you saying that *neither* TS or HTML changes are reflected in the browser? I can't reproduce that, I see changes appearing in the browser. Do you always get this?

**Comment by hansl** [comment]
This should have been fixed by #5492.

**Comment by etovian** [comment]
@filipesilva HTML was working fine.  I was just demonstrating that the problem was limited to TypeScript.  @hansl What version will the #5492 fix release in?  Thanks!
